pyproject.toml

```diff
@@ -35,7 +35,7 @@ build-backend = "poetry.masonry.api"
 
 [tool.poetry.dependencies]
 python = "^3.8"
-graphql-core = "~3.2.0"
+graphql-core = {git = "https://github.com/graphql-python/graphql-core.git", branch = "main"}
```

Comment:
I'm ok with adding support to 3.3 now, but could we test both 3.2 and 3.3 using nox? 😊

Reply:
yeah sure

Reply:
https://github.com/strawberry-graphql/strawberry/actions/runs/9928799671/job/27425551808?pr=3570#step:6:7838

Most of the tests won't pass due to change in the formatter.
How do you think we can overcome this?

Reply:
we can force those tests to only run on the latest version of graphql core 😊